specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-06-01-preview/SourceControls.json

```diff
@@ -213,18 +226,20 @@
             }
           }
         }
-      },
-      "put": {
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/sourcecontrols/{sourceControlId}/delete": {
+      "post": {
```

Comment:
>post [](http://example.com/codeflow?start=7&length=4)

Delete operation as POST?

Reply:
We need to pass `body` into our delete operation. As custom headers are not supported this is the suggested way I got during Office Hours.

Reply:
There is no body in request payload now, as I can see. Should this be Delete operation now or is there any reason for it to be a POST Action?